api/src/test/java/org/pmiops/workbench/utils/TestMockFactory.java

```diff
@@ -405,6 +408,29 @@ public static LeonardoListPersistentDiskResponse createLeonardoListRuntimePDResp
         pdName, status, date, googleProjectId, user, /*appType*/ null);
   }
 
+  public static Disk newDisk(
```

Comment:
This works as-is.

But since we have diverging behavior for the 2 cases (app vs. runtime) maybe it sense to separate these into newAppDisk and newRuntimeDisk?  your choice.

Reply:
I think that is fair. I restructured it and kept the common code as a shared private function. I also renamed it createXDisk.